.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Create Release
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+    contents: write
+
+jobs:
+  release:
+    runs-on: windows-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Setup .NET SDK 8.0
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: 8.0.x
+    - name: Build
+      run: dotnet build src\DurangoInteropDotnet.csproj -property:Configuration=Release
+    - name: Create Release
+      uses: actions/create-release@v1
```

Comment:
Please use https://github.com/softprops/action-gh-release instead of unmaintained actions/create-release, even if it still seems to work.

Reply:
Thanks for the pointer, I've never messed with github actions.

Using my forked repo I was able to utilize `softprops/action-gh-release@v2` to package & publish a release whenever a tag is pushed